chrome/content/nostalgy.js

```diff
@@ -280,18 +279,31 @@ function NostalgyHide(restore) {
 function NostalgyDefLabel() {
  nostalgy_gsuggest_folder = NostalgySuggest();
  if (nostalgy_gsuggest_folder) {
-   nostalgy_label.value =
-       nostalgy_default_label + " [+Shift: ==> " + NostalgyFolderName(nostalgy_gsuggest_folder) + "]";
+   nostalgy_label.label =
+       nostalgy_default_label + " [+Shift: => " + NostalgyFolderName(nostalgy_gsuggest_folder) + "]";
  } else {
-   nostalgy_label.value = nostalgy_default_label;
+   nostalgy_label.label = nostalgy_default_label;
  }
 }
 
 
 function NostalgyCollapseFolderPane() {
  var fp = NostalgyEBI("folderPaneBox");
  if (window.MsgToggleFolderPane)
-   { MsgToggleFolderPane(true); return true; }
+   { 
+/*  var splitter = document.getElementById("folderpane_splitter");
+  var state = splitter.getAttribute("state");
+  if (state == "collapsed") {
+    splitter.setAttribute("state", "open");
+  } else {
+    splitter.setAttribute("state", "collapsed");
+    }
+*/
+    //   fp = NostalgyEBI("folderpane_splitter");
+  // fp.state="collapsed";
+   MsgToggleFolderPane();
+ //  goDoCommand('cmd_toggleFolderPane'); 
+   return true; }
```

Comment:
I suggest removing the commented out code.

If I see this correctly, the actual change boils down to `MsgToggleFolderPane(true)` ➡️ `MsgToggleFolderPane()` .